### DISCUSSION TITLE
[Bug report] Footer socials media links not redirecting properly

### DISCUSSION THREAD

**Discussion by TabiZz** [GetPublii/Publii#1629] [Bug Report] [answered]
### Operating system

Sonoma 14.6.1 (23G93)

### Publii version

0.46.0 (build 16910)

### Issue type

Application

### Bug description

Hey, on my footer, I put my socials media link. When I click on the logo on my footer it redirect to **https://[Website]/[Page Name]/facebook.com/xxx** instead of just **facebook.com/xxx**

Same for Instagram and I think other social media.

<img width="1139" alt="Screenshot 2024-08-21 at 15 16 41" src="https://github.com/user-attachments/assets/57d23985-2993-4935-b563-e7600ca0bb1d">

<img width="548" alt="Screenshot 2024-08-21 at 15 17 26" src="https://github.com/user-attachments/assets/1f7a3667-ad1e-4cfb-a7ed-77d5d8d3d45d">

Any help on that subject?

**Comment by bobmitro** [comment]
The Publii app does not generate these links (icons) as they are handled by the theme. Which theme are you using, and have you made any custom changes? Please provide a link to the page where the issue appears so that we can assist you.

**Reply by TabiZz** [reply]
My bad, so it's theme related issue and not a Publii issue.

Using Simple 3.0.0
This is my website : https://creationsdulizieux.com/

<img width="980" alt="image" src="https://github.com/user-attachments/assets/04ad7147-8e6b-45b6-afb2-5c3ef7e96a91">

**Comment by bobmitro** [comment, marked as answer]
You must provide the full URL, including "https://" at the beginning. So, instead of `facebook.com/creationsdulizieux`, enter `https://www.facebook.com/creationsdulizieux`.

**Reply by TabiZz** [reply]
Oh nice catch! Gotcha, working now
